azext_iot/iothub/commands_certificate.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import Optional
+from azext_iot.iothub.providers.certificate import CertificateProvider
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def certificate_root_authority_show(
+    cmd,
+    hub_name: Optional[str] = None,
+    resource_group_name: Optional[str] = None,
+    login: Optional[str] = None,
+    auth_type_dataplane: Optional[str] = None
+):
+    resource = CertificateProvider(
+        cmd=cmd,
+        hub_name=hub_name,
+        rg=resource_group_name,
+        login=login,
+        auth_type_dataplane=auth_type_dataplane
+    )
+    return resource.iot_hub_certificate_root_authority_show()
+
+
+def certificate_root_authority_set(
+    cmd,
+    ca_version: str,
+    hub_name: Optional[str] = None,
+    resource_group_name: Optional[str] = None,
+    login: Optional[str] = None,
+    auth_type_dataplane: Optional[str] = None,
+    yes: Optional[str] = False
```

Comment:
Optional typing means the explicit value of None is supported. Also, what is here is Optional typing of a str, where-as the default assigned value is bool and the iot_hub_certificate_root_authority_set type hint is straight bool

![image](https://user-images.githubusercontent.com/28658112/186040283-96780dd0-d727-42d2-8fe5-25c4e080ca25.png)

So is this typing declaration the most accurate usage?

Reply:
typo